docs/service-bundle.md

```diff
@@ -0,0 +1,225 @@
+# Service Bundle Specification
+
+A Service Bundle is a container image that the broker can use to manage
+deployment of a service in a cluster. It adheres to the properties and
+behaviors described below.
+
+## Label
+
+Every service bundle container image has a label ‘com.redhat.apb.spec’ that
```

Comment:
Has anyone asked the question: Is an APB considered one of many types of service bundles that simply adheres to the contract, and chooses to implement its behavior in Ansible? I think this label may need to be more generic.

Reply:
```bundle.spec```?

Reply:
Yeah perhaps worth removing `redhat` too eh? Label could just become `bundle.spec`.

Reply:
I know that @mhrivnak and I had discussed this and the answer was yes. An APB is an instance of a Service Bundle adhering to the contract.

It wasn't clear to me, @rthallisey if you are suggesting the label be `com.redhat.bundle.spec` or just `bundle.spec`. Only ask since we already have `com.redhat.apb.runtime` (which would probably remain since it is APB specific).

Reply:
Only ```bundle.spec```

Reply:
Hey guys, I think that this is a follow-up issue:

> Please review how accurately this represents the current relationship between a Service Bundle and the Automation Broker as implemented today. Please do not (yet) review the technical merits of this design.

A discussion that we need to have I just don't think that this is the right place just yet IMO